src/gurt/tests/test_gurt_telem_consumer.c

```diff
@@ -199,6 +205,41 @@ test_timer_snapshot(void **state)
 	assert((tms3.tv_sec + tms3.tv_nsec) > 0);
 }
 
+static void
+test_gauge_stats(void **state)
+{
+	struct d_tm_stats_t	stats;
+	uint64_t		val;
+	int			rc;
+
+	rc = d_tm_get_gauge(&val, &stats, shmem_root, NULL,
+			    "gurt/tests/telem/gauge-stats");
+	assert(rc == D_TM_SUCCESS);
+
+	assert(val == 20);
+	assert(stats.dtm_min.min_int == 2);
+	assert(stats.dtm_max.max_int == 20);
+	assert(fabs(stats.mean - 11.0) < 0.1);
+	assert(fabs(stats.std_dev - 5.89379) < 0.00001);
```

Comment:
Could use the cmocka macro assert_float_equal, makes it a little clearer. There's also an assert_int_equal for the other cases.

Reply:
Thank you.  I didn't know about assert_float_equal.  I'll switch that over provided they work as expected.